.devcontainer/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# From https://github.com/microsoft/vscode-dev-containers/blob/master/containers/go/.devcontainer/Dockerfile
+ARG VARIANT=1
+FROM mcr.microsoft.com/vscode/devcontainers/go:${VARIANT}
+
+# [Optional] Install a version of Node.js using nvm for front end dev
+ARG INSTALL_NODE="true"
```

Comment:
This repo doesn't have a front end component. Not sure everyone would want to opt for this.

Reply:
It is actually disabled by the option in devcontainer.json. I don't know why they did it like that, and I preferred to do straight copypasta so updating from the source listed up top would be easier.